Fuse/qlack2-fuse-auditing/qlack2-fuse-auditing-impl/src/main/java/com/eurodyn/qlack2/fuse/auditing/impl/model/Audit.java

```diff
@@ -31,7 +32,7 @@
 	@ManyToOne(fetch = FetchType.LAZY)
 	@JoinColumn(name = "level_id")
 	private AuditLevel levelId;
-	@ManyToOne(fetch = FetchType.LAZY)
+	@OneToOne(fetch = FetchType.LAZY)

```

Comment:
Nassos to comment. We are changing the behaviour of the audit and audit trace relation. Even though it is used now one-to-one, it might have started differently in the initial requirements.